packages/graphql-types/types/Account.graphql

```diff
@@ -267,6 +267,14 @@ interface AbstractLoanAccount {
   originationDate: Date
   payoff: Money
   secured: Boolean
+  paymentOptions: [LoanPaymentOptions]
+}
+
+type LoanPaymentOptions {
+  paymentType: PaymentType!
+  defaultValue: Money
+  value: Money
+  max: Money
```

Comment:
Should I model this out in a more sophisticated manner? i.e., only allow certain fields to be available depending on which payment type is selected? 

I know that graphql supports union types which would help with this. But I couldn't see a way to use "literal" types like in typescript to help with this.

Reply:
I think we're good with this -- no need to get fancier at the moment. Thanks!